lib/manageiq/password.rb

```diff
@@ -158,8 +162,11 @@ def self.load_key_file(filename)
       Key.new(*YAML.load_file(filename).values_at(:algorithm, :key, :iv))
     end
 
-    def self.extract_erb_encrypted_value(value)
-      return $1 if value =~ /\A<%= (?:MiqPassword|DB_PASSWORD|ManageIQ::Password)\.decrypt\(['"]([^'"]+)['"]\) %>\Z/
+    def self.remove_erb(str)
+      return str if str.nil? || str.empty? || !str.start_with?("<%=")
```

Comment:
I added one extra check here for the string starting with `"<%="`, which is faster than a regex, and since the non-erb case is the 99% case, this should basically eliminate any actual regex checks here.